app/components/adjust-recipe/template.hbs

```diff
@@ -0,0 +1,22 @@
+{{#each recipe as |recipe|}}
```

Comment:
You can't have a block variable named the same as the array you are looping over, I recommend changing to `recipes` and `recipe`. Note that you'll need to update your `recipe/template.hbs` file to pass in proper data.